radicle-node/src/service.rs

```diff
@@ -1139,6 +1152,56 @@ where
         )
     }
 
+    // XXX: does not take &self to appease the borrow checker
```

Comment:
Hmm, should it be on `tracking::Config` maybe? `Config::namespaces_for(rid, storage)`

Reply:
Ya, that sounds good! Nice suggestion :)

Reply:
Fixed that up and rebased with the `FetchDirection` changes :)